service/storage_service.cc

```diff
@@ -2655,7 +2661,8 @@ future<node_ops_cmd_response> storage_service::node_ops_cmd_handler(gms::inet_ad
                 }
                 return update_pending_ranges(tmptr, format("removenode {}", req.leaving_nodes));
             }).get();
-            auto meta = node_ops_meta_data(ops_uuid, coordinator, std::move(req.ignore_nodes), [this, coordinator, req = std::move(req)] () mutable {
+            auto watchdog_interval = watchdog_timeout(_db.local().get_config());
```

Comment:
If we pass db config to node_ops_meta_data then it would depend on the db config. I didn't want to add this dependency, seem too low level an object to depend on db.

Reply:
Yes, but it's a preexisting problem, we already repeat `_node_ops.emplace(ops_uuid, std::move(meta));`. I can add a cleanup if you insist.

Reply:
yes please, I think it will make maintenance of this code easier.